src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -557,48 +564,74 @@ public JSONObject getDebugState()
         return debugState;
     }
 
+
     /**
-     * A terminating {@link Node}, which sends the packets through a
-     * datagram socket.
+     * A {@link ConsumerNode} which sends a packet out over a socket
      */
     private class SocketSenderNode extends ConsumerNode
```

Comment:
This class, the way it's implemented here, is incorrect, I think, and should be tweaked.  This is implemented as a `ConsumerNode` but it does *not* take ownership of the buffer, as it doesn't free it.  The issue was that we wanted to return SRTP packets to the pool but not those coming from the DTLS pipeline.  In fact, I think this change may be defunct now because I started copying outgoing DTLS packets into pooled buffers, so this may currently be a leak (which just means we can go back to using a single sender for both DTLS and SRTP, which would be better anyway).  I'll look into this.

Reply:
Ok, confirmed that this is no longer necessary (since we now copy outgoing SCTP buffers into pooled buffers).